library-best-practices/JLBP-11.md

```diff
@@ -0,0 +1,7 @@
+[JLBP-11] Frequent release cadence for non-breaking changes
+-----------------------------------------------------------
+
+- Keep compatible dependencies up-to-date.
+- Recommendation: Perform a release at least once every 6-weeks.
+  - Don't be more than 6 weeks out of date with your dependencies.
```

Comment:
Perhaps measure in versions rather than time? One minor release?

Reply:
I think the time dimension is important, because having the ecosystem generally up to date at each point in time results in ecosystem snapshots that are more consistent, and more likely to have compatibility across the board.